recipes/sdjson/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "sdjson" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sdjson-{{ version }}.tar.gz
+  sha256: d15e54d3727ee6b175014494a31a2bd6012da3cd3083850ba9cfb58aef0c4f3b
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - whey >=0.0.20
```

Comment:
This is not restricted upstream. Since you're upstream as well, can you please sync this? it's fine for now but should be corrected at some point.